docs/issue.adoc

```diff
@@ -0,0 +1,31 @@
+:page-title: @Issue
+:page-description: JUnit Jupiter extensions to add information about a related issue
+
+== Introduction
+
+The main reason tests are written is to ensure the functionality of a requirement or to avoid technical problems.
+The `@Issue` allows to mark tests with the related issue, like a requirement or a bugfix, of an issue-tracker like JIRA or Redmine.
```

Comment:
What do you think of this:
"The `@Issue` annotation allows marking tests with a String, referencing a related issue (like a requirement or a bugfix) of an issue-tracker (like JIRA or Redmine)."